src/env.example.json

```diff
@@ -0,0 +1,6 @@
+{
+  "api": {
+    "protocol": "http",
+    "host": "127.0.0.1:3000"
+  }
+}
```

Comment:
Is this still being used?

Reply:
@junthehacker fixed in e0bfa96cb1bb8b04aefa9a19cb96cac89fd1ef35